Update Language Section

zanbaldwin opened the pull request:
Add paragraph about the use of pronouns when talking to or about an individual person.

ciaranmcnulty commented on Guideline.md:
```diff
@@ -62,7 +62,11 @@ Knowing your audience is critical to ensuring that meeting room layouts, menus,
 
 #### Avoid gendered pronouns
-Avoid using male pronouns ("he", "him", "his") in circumstances that describe people of any gender. Find [examples of nonsexist language](http://geekfeminism.wikia.com/wiki/Nonsexist_language). 
+Avoid using male pronouns ("he", "him", "his") in circumstances that describe people of any gender. Find [examples of nonsexist language](http://geekfeminism.wikia.com/wiki/Nonsexist_language).
```
This reads like 'avoid using male pronouns [...] for anyone' which conflicts with the latter advice about using preferred pronouns

I think by 'people of any gender' you mean 'people who could be any gender'?

zanbaldwin replied:
I'm under the assumption that the paragraph at line 65 is talking about (two or more) *people*, while I'm adding paragraphs at line 67-69 to talk about *individuals*.

I didn't want to change the original paragraph (the diff is only whitespace) unless I had some feedback and knew how to better clarify meaning. Once I have that I can attempt to improve grammar flow between the paragraphs.

zanbaldwin replied:
```diff
-Avoid using male pronouns ("he", "him", "his") in circumstances that describe people of any gender.
+Avoid using male pronouns ("he", "him", "his") as a collective term in circumstances that describe people of any, or multiple, genders.
```

?

zanbaldwin committed:
Update Language Section

Add paragraph about the use of pronouns when talking to or about an individual person.

zanbaldwin commented on Guideline.md:
```diff
@@ -62,7 +62,11 @@ Knowing your audience is critical to ensuring that meeting room layouts, menus,
 
 #### Avoid gendered pronouns
-Avoid using male pronouns ("he", "him", "his") in circumstances that describe people of any gender. Find [examples of nonsexist language](http://geekfeminism.wikia.com/wiki/Nonsexist_language). 
+Avoid using male pronouns ("he", "him", "his") as a collective term in circumstances that describe people of any, or multiple, genders. Find [examples of nonsexist language](http://geekfeminism.wikia.com/wiki/Nonsexist_language).
```
```diff
-as a collective term in circumstances that describe people of any, or multiple, genders.
+as a collective term in circumstances that describe multiple people of any, or differing, genders.
```

Thoughts?
